servers/rendering/renderer_scene_cull.cpp

```diff
@@ -3376,6 +3390,23 @@ RID RendererSceneCull::_render_get_environment(RID p_camera, RID p_scenario) {
 	return RID();
 }
 
+RID RendererSceneCull::_render_get_compositor(RID p_camera, RID p_scenario) {
+	Camera *camera = camera_owner.get_or_null(p_camera);
+	if (camera && scene_render->is_compositor(camera->compositor)) {
+		return camera->compositor;
+	}
+
+	Scenario *scenario = scenario_owner.get_or_null(p_scenario);
+	if (!scenario) {
+		return RID();
+	}
+	if (scene_render->is_compositor(scenario->compositor)) {
```

Comment:
```suggestion
	if (scenario && scene_render->is_compositor(scenario->compositor)) {
```
Just a minor suggestion

Reply:
looks good :)